### ISSUE TITLE
Toolkit GUI accepts commas in lat/long field which leads to JSON arrays

### ISSUE THREAD

**Issue by arlake228** [perfsonar/toolkit#359]
See report from user:

> toolkits allow entering commas(?) in latitude/longitude fields, so that they show as an Array in the JSON, versus a scalar.
> reference:
> http://192.41.236.36/toolkit
> http://128.120.218.218/toolkit/

**Comment by bhavesh37** [comment]
@arlake228 Looks like the form does validate correctly for latitude and longitude on the dev box I'm working on(perfsonar-dev9). Please have a look at the attached image:

![screen shot 2019-02-21 at 4 15 38 pm](https://user-images.githubusercontent.com/24940776/53202694-2aa18980-35f5-11e9-88e0-ceb8424faba8.png)

Is the issue related to something else? Maybe, the user changed configs for the form on his toolkit?

**Comment by arlake228** [comment]
I also cannot recreate at the web interface level. I can recreate the problem if I add lines like the following manually to /etc/perfsonar/lsregistrationdaemon.conf:

```
latitude   70
latitude
```

The reason is that the file is in Config::General format and by default perl converts values specified multiple times to an array. Input like this actually fails in the LSRegistrationDaemon. For example, I got this in the lsregistrationdaemon.log  when I tried the above:

```
2019/03/06 15:57:23 (6780) ERROR> Base.pm:158 perfSONAR_PS::LSRegistrationDaemon::Base::validate_conf - Multiple entries for latitude. Only one is valid
```

So it's bad input that is not possible from the UI anymore (probably was at one time, hence these hosts) that isn't getting registered to the LS. The only issue is that it is working its way into the host JSON.

Probably the short-term fix is to ignore that field unless it is a scalar in the host.cgi script. Longer-term we should probably consolidate validation of that file as I am sure there are other issues possible like this.
